csrc/executor.h

```diff
@@ -179,26 +179,89 @@ class FusionExecutor : public NonCopyable {
     return measure_kernel_time_ ? kernel_time_ms_ : 0;
   }
 
+  //! Returns the input bytes accessed for a kernel
+  //! \note It is important to sample the args struct prior to adding the
+  // 1    output to the args struct
+  int64_t inputBytesProcessed(KernelArgumentHolder& args) {
```

Comment:
Can this be a `const KernelArgumentHolder&`?

Reply:
Made the change.